src/Tribe/Views/V2/View.php

```diff
@@ -806,12 +819,18 @@ public function prev_url( $canonical = false, array $passthru_vars = [] ) {
 		}
 
 		// Make sure the view slug is always set to correctly match rewrites.
-		$url = add_query_arg( [ 'eventDisplay' => $this->slug ], $url );
+		$query_args['eventDisplay'] = $this->slug;
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: f35a3c4b89eca3aafc46c15dd13ca746</sub>